drivers/gles3/storage/particles_storage.h

```diff
@@ -158,6 +158,7 @@ class ParticlesStorage : public RendererParticlesStorage {
 		int amount = 0;
 		double lifetime = 1.0;
 		double pre_process_time = 0.0;
+		real_t request_process_time = 0.0;
```

Comment:
Shouldn't this be `double`, like `pre_process_time`?

Reply:
I feel like it's really unnecessary to use a double for that